addons/sales_team/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import test_default_team
+__all__ = ['test_default_team']
```

Comment:
Shouldn't be needed

Reply:
Well that was added to just be compliant with pep8 (technically yes, it is not needed). Though, if you don't care about that, I can remove it.

Reply:
please remove, we are not pep8 aficionados and I'd rather be consistent with the rest of the repo which never specifies explicit exports

Reply:
Removed. Though, in our projects we try to follow pep8, but I agree that specific project's guidelines takes priority, so no problem here:)